app/src/forms/admin/service.js

```diff
@@ -116,7 +118,180 @@ const service = {
         email: fa.email,
         roles: fa.roles
       }));
-  }
+  },
+
+  /**
+   * @function createFormComponentsProactiveHelp
+   * insert each Form Component Help Info
+   * @param {Object} data Form Component Help Info object
+   * @returns {Promise} An objection query promise
+   */
+  createFormComponentsProactiveHelp: async(data)=> {
+    let trx;
+    try {
+
+      trx = await FormComponentsProactiveHelp.startTransaction();
+
+      let id = data&&data.componentId;
+
+      let buf, imageType;
+      if(data.image!==''){
+        buf = data.image.split(',')[1];
+        imageType = (data.image.split(';')[0]).split(':')[1];
+      }
+
+      if(id) {
+        await FormComponentsProactiveHelp.query(trx).patchAndFetchById(data.componentId, {
+          componentname : data&&data.componentName,
```

Comment:
The same as in the model, the column names should be camel cased instead of lower case to stay consistent with the rest of the application.